lib/resque/plugins/red_cross_async.rb

```diff
@@ -3,27 +3,53 @@ module Plugins
     module RedCrossAsync
       # Hash of properties for RedCross.monitor
       # Please add count: MY_NUMBER in case your metric value differs 1
+      mattr_accessor :job_start_time, :job_end_time
+
       def properties(*args)
         {}
```

Comment:
@dmitri86git as discussed this should have a filter function for the args hash to make sure it will send only specific parameters.